src/App.vue

```diff
@@ -4,6 +4,9 @@
   <GlobalDialog />
   <GlobalToast />
   <UnloadWindowConfirmDialog />
+  <Teleport to="body">
```

Comment:
Why do we want to teleport the AppMenu to body? Do do want to allow the menu to float over the sidebar panel if the sidebar panel is resized big enough?